src/pages/studyView/StudyViewPageStore.ts

```diff
@@ -350,6 +354,41 @@ export class StudyViewPageStore {
             )
         );
 
+        this.reactionDisposers.push(
+            reaction(
+                () => this.sharedGroupSet,
+                groups => {
+                    if (Object.keys(groups).length > 0) {
+                        this.showGroupsTooltip = true;
```

Comment:
by groupsTooltip do we mean the whole groups UI dropdown?   If so, I think we should call it comparisonGroupUI or something.   tooltip suggests something less significnat.